doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1566,7 +1567,7 @@ It must follow the same representation conventions as context attribute metadata
 A `subject` contains the following subfields:
 
 + `entities`: A list of objects, each one composed of the following subfields (`id`/`idPattern` or type must be present):
-    + `id` or `idPattern`: Id or pattern of the affected entities (optional). Both cannot be used at the same time.
+    + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same time, but at least one of them must be set.
```

Comment:
Two occurrence in `query` and `discover` operations, but, these cases have other behavior:
>  `id` or `idPattern` (both cannot be used at the same time): specifies the exact ID (in the case of `id`)
     or pattern (in the case of `idPattern`) of the entities to search for. If omitted, it means "any entity
     ID".

Other ocurrense is in `list entities` but in this case both field are optionals